### PR TITLE
Fix cli auto completion

### LINKED ISSUE #8481: Trino CLI prints exceptions to stdout when pressing Tab for completion
Since version 359:

```sql
# Press TAB after any input
trino> SHOW TAJul 06, 2021 11:35:35 PM org.jline.utils.Log logr
INFO: Error while parsing line
org.jline.reader.EOFError
	at io.trino.cli.InputParser.parse(InputParser.java:46)
	at org.jline.reader.impl.LineReaderImpl.doComplete(LineReaderImpl.java:4389)
	at org.jline.reader.impl.LineReaderImpl.doComplete(LineReaderImpl.java:4365)
	at org.jline.reader.impl.LineReaderImpl.expandOrComplete(LineReaderImpl.java:4304)
	at org.jline.reader.impl.LineReaderImpl$1.apply(LineReaderImpl.java:3798)
	at org.jline.reader.impl.LineReaderImpl.readLine(LineReaderImpl.java:670)
	at org.jline.reader.impl.LineReaderImpl.readLine(LineReaderImpl.java:498)
	at org.jline.reader.impl.LineReaderImpl.readLine(LineReaderImpl.java:484)
	at io.trino.cli.InputReader.readLine(InputReader.java:59)
	at io.trino.cli.Console.runConsole(Console.java:239)
	at io.trino.cli.Console.run(Console.java:191)
	at io.trino.cli.Console.call(Console.java:102)
	at io.trino.cli.Console.call(Console.java:75)
	at picocli.CommandLine.executeUserObject(CommandLine.java:1953)
	at picocli.CommandLine.access$1300(CommandLine.java:145)
	at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2352)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2346)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2311)
	at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2179)
	at picocli.CommandLine.execute(CommandLine.java:2078)
	at io.trino.cli.Trino.main(Trino.java:42)
```

This seems to be introduced in https://github.com/trinodb/trino/commit/dabd7fe45f. It's the first commit that can reproduce this issue.

cc: @electrum

### PR DESCRIPTION
**Pull request by lucasdlemos**
- The cli auto completion feature was silently broken for a while and, recently, started printing an error to stdout when an user presses `tab`.
- This PR fixes it by handling the `COMPLETE` ParseContext
- Additionally fixes completion for commands with multiple words
- Fixes #8481 and #8495

### CODE REVIEW

File: client/trino-cli/src/main/java/io/trino/cli/Completion.java [hunk 1]
```diff
@@ -13,42 +13,42 @@
...
 
     public static Completer commandCompleter()
     {
-        return new StringsCompleter(ImmutableSet.<String>builder()
-                .addAll(COMMANDS)
-                .addAll(COMMANDS.stream()
-                        .map(value -> value.toLowerCase(ENGLISH))
-                        .collect(toSet()))
-                .build());
+        return new AggregateCompleter(buildArgumentCompleter("CREATE", singletonList("TABLE")),
```
Comment (kokosing) [inline client/trino-cli/src/main/java/io/trino/cli/Completion.java:33]: Can we have also SCHEMA? Can we also have other statements too like `ALTER TABLE/SCHEMA`

**Review by kokosing — COMMENTED** [review]
Thanks

File: client/trino-cli/src/main/java/io/trino/cli/Completion.java (hunk 1 above)
Comment (lucasdlemos) [reply client/trino-cli/src/main/java/io/trino/cli/Completion.java]: I initially kept the list of commands as it was before, but sure. I added those you suggested.

**Review by wendigo — APPROVED** [review]

**Review by kokosing — APPROVED** [review]

**Comment by martint** [comment]
@cla-bot check

**Comment by lcx412** [comment]
The exceptions still show in version 360.

**Comment by hashhar** [comment]
@lcx412 make sure you are Trino CLI version >= 360. This was not a change in Trino server and the server version doesn't matter for this bug.

If you can still reproduce with Trino CLI >= 360, please share the stack trace/output so we can debug.

**Comment by lcx412** [comment]
@hashhar 
Sorry, I just upgraded Trino server version to 360 and forgot the CLI version.
All good now. Thanks.

**Pull request merged**